src/devices/bus/pce/pce_rom.cpp

```diff
@@ -1,5 +1,5 @@
 // license:BSD-3-Clause
-// copyright-holders:Fabio Priuli
+// copyright-holders:Fabio Priuli, Charles MacDonald, Wilbert Pol, Angelo Salese
```

Comment:
Arcade Pro stuff should go to a separate file, and credited to me only (since it still uses the old and outdated i/f), likewise I think the base Super CD system is mostly a @wilbertpol product?

Regardless, considering it eventually needs Expansion bus bindings I believe having both as standalone files is for the best.